src/renpy_warp.rpe.py

```diff
@@ -15,8 +15,13 @@
 
 import renpy  # type: ignore
 
-enabled = bool(os.getenv("WARP_ENABLED"))
-port = os.getenv("WARP_WS_PORT")
+ENABLED = bool(os.getenv("WARP_ENABLED"))
+PORT = os.getenv("WARP_WS_PORT")
+NONCE = os.getenv("WARP_WS_NONCE")
+
+logHere = open("/tmp/socketLog/log.txt", mode="w", buffering=1)
```

Comment:
do we want to keep this? this breaks if `/tmp/socketLog/` doesn't already exist

Reply:
i'm also worried about cross platform compatibility, and not closing the file after you've opened it for writing

Reply:
That wasn't supposed to have been committed... 🤦I thought I stashed that sorry. Thanks for spotting it.